CentralComputing/UnitTests/StateMachine_Test.cpp

```diff
@@ -1,53 +1,68 @@
-#include "Sensor.h"
-#include <iostream>
-#include <unistd.h>
-#include "Pod.h"
-#include "StateMachineCompact/stdafx.h"
-#include <string>
-#include <fstream>
-#include <sstream>
-#include <algorithm>
-#include <vector>
-#include <stdlib.h>
+#include "StateMachine_Test.h"
 
-#include <time.h>
+//Include any other dependencies 
 
-using namespace std;
-
-// declaring function, implented later
-void print_help();
-void print_help_instr();
+StateMachine_Test::StateMachine_Test(){
```

Comment:
I see what you are saying. The idea that I had behind the Unit Tests were that they were encapsulated tests that would do everything they needed to do when the call to test() was made. Essentially, a call to test() is a call to main(), and the test can do anything it wants. 

Even though the full pod test suite would indeed be monolithic, it would be monolithic anywhere. The proposed Unit Test layout provides a common interface for all tests. Want to run a short test on the thermocouples? Type `./build_test thermo`. Want to run the big StateMachine test? Run `./build_test statemachine`. If needed, the StateMachine_test class can depend on other classes, **but I think it makes sense to keep it linked to the common testing interface**